cassandra-sink/src/main/java/org/springframework/cloud/stream/module/cassandra/CassandraProperties.java

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.module.cassandra;
+
+import javax.validation.constraints.AssertFalse;
+import javax.validation.constraints.NotNull;
+
+import org.hibernate.validator.constraints.Range;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.cassandra.config.CassandraCqlClusterFactoryBean;
+import org.springframework.cassandra.config.CompressionType;
+import org.springframework.util.StringUtils;
+
+/**
+ * Properties for the cassandra sink.
+ *
+ * @author Artem Bilan
+ * @author Thomas Risberg
+ */
+@ConfigurationProperties("spring.cassandra")
+public class CassandraProperties {
+
+	/**
+	 * The comma-delimited string of the hosts to connect to Cassandra.
+	 */
+	private String contactPoints = CassandraCqlClusterFactoryBean.DEFAULT_CONTACT_POINTS;
+
+	/**
+	 * The port to use to connect to the Cassandra host.
+	 */
+	private int port = CassandraCqlClusterFactoryBean.DEFAULT_PORT;
+
+	/**
+	 * The keyspace name to connect to.
+	 */
+	@Value("${spring.application.name:tmp}")
+	private String keyspace;
+
+	/**
+	 * The username for connection.
+	 */
+	private String username;
+
+	/**
+	 * The password for connection.
+	 */
+	private String password;
+
+	/**
+	 * The path to file with CQL scripts (delimited by ';') to initialize keyspace schema.
+	 */
+	private String initScript;
```

Comment:
I found that this one can be specified as a `Resource`.
Then in the debug I see this result:
```
result = {DefaultResourceLoader$ClassPathContextResource@7696} "class path resource [init-db.cql]"
 path = "init-db.cql"
 classLoader = {Launcher$AppClassLoader@7699} 
 clazz = null
```

Let me know if we can go ahead with the `Resource` approach and I'll fix it on merge.
Otherwise explain, please, what is the concern exists do not do that.
Thanks